deploy/executor-release.yaml

```diff
@@ -3,4 +3,17 @@ steps:
   - name: "gcr.io/cloud-builders/docker"
     args: ["build", "-f", "deploy/Dockerfile",
            "-t", "gcr.io/kaniko-project/executor:${COMMIT_SHA}", "."]
-images: ["gcr.io/kaniko-project/executor:${COMMIT_SHA}"]
+  # Then, we want to clone the distroless repo and build busybox.tar
+  - name: "gcr.io/cloud-builders/git"
+    args: ["clone", "https://github.com/GoogleContainerTools/distroless.git"]
+  - name: "gcr.io/cloud-builders/bazel"
+    args: ["build", "busybox:busybox_tar"]
```

Comment:
Hmm, could we do this in another stage of the dockerfile? otherwise we won't be able to build locally.

Reply:
Oh yah that makes sense, just added it!